shared/actions/chat2/index.tsx

```diff
@@ -3366,8 +3366,10 @@ const onUpdateLastCoord = async (action: Chat2Gen.UpdateLastCoordPayload) => {
 
 const openChatFromWidget = ({payload: {conversationIDKey}}: Chat2Gen.OpenChatFromWidgetPayload) => [
   ConfigGen.createShowMain(),
-  RouteTreeGen.createSwitchTab({tab: Tabs.chatTab}),
```

Comment:
navigateToThread sets things up correctly, you don't need to switch (which loses the params temporarily which is annoying)